config/shredder.stream.config.reference.hocon

```diff
@@ -74,6 +74,13 @@
     ]
   },
 
+  "options": [
```

Comment:
I wonder if this should be an object instead of an array

```
  "options": {
    "minimumTimestamp": "2021-11-18T11:00:00.00Z"
  }
```

I say this because [the lightbend config library](https://github.com/lightbend/config) has a very awkward syntax for setting array values from system properties and environment variables, so I often think it's not worth fighting with arrays.

I also wonder if "filters" or "validations" is a better word than "options".

Reply:
Agree, that would be better. I will update it 👍